### PR TITLE
[feat lang]: Chinese readme (half-way)

### PR DESCRIPTION
**Pull request by seognil**
## Description
Hi, there,

I wrote a Chinese Version of the readme to let more people saw your great work 🎉  
(by add a `README_SOURCE-zh.md`)  
But I'm not sure the `doctoc`, so **other files are untouched**

Is there any chance for us to finish the process and merge it?

## Changed
- [x] words description translated line by line
- [ ] code block untouched (maybe working on comments later)  
- [x] toc untouched but auto generated
- [ ] contributors section untouched
- [x] `back to top` and some other in-place anchors translated
- [x] some docs links translated to zh version (tslang.cn & zh-hans.reactjs.org)

## Checklist

* [x] I have read [CONTRIBUTING.md](https://github.com/piotrwitek/react-redux-typescript-guide/blob/master/CONTRIBUTING.md)
* [ ] I have edited `README_SOURCE.md` (NOT `README.md`)
* [ ] I have run CI script locally `npm run ci-check` to generate an updated `README.md`
* [ ] I have linked all related issues above
* [ ] I have rebased my branch

### CODE REVIEW

**Comment by piotrwitek** [comment]
Hey @seognil.
Oh man, that's really awesome thanks for your incredible work.
I'm really excited about it and I would love to add to the repository.

For now, let's try to merge it ASAP, I'll try to review it right now.

For the near future, I'll think about some more generic way using internationalization techniques to make it easier to improve and maintain the translations.

File: README_SOURCE-zh.md [hunk 1]
```diff
@@ -0,0 +1,994 @@
...
+    - [**目标**](#%e7%9b%ae%e6%a0%87)
+    - [**React、Redux、Typescript 生态系统**](#reactreduxtypescript-%e7%94%9f%e6%80%81%e7%b3%bb%e7%bb%9f)
+    - [**示例**](#%e7%a4%ba%e4%be%8b)
+    - [**Playground 项目**](#playground-%e9%a1%b9%e7%9b%ae)
+  - [贡献指南](#%e8%b4%a1%e7%8c%ae%e6%8c%87%e5%8d%97)
+  - [赞助](#%e8%b5%9e%e5%8a%a9)
+  - [目录](#%e7%9b%ae%e5%bd%95)
+- [安装](#%e5%ae%89%e8%a3%85)
+    - [React & Redux 的类型定义](#react--redux-%e7%9a%84%e7%b1%bb%e5%9e%8b%e5%ae%9a%e4%b9%89)
+- [React - 类型定义速查表](#react---%e7%b1%bb%e5%9e%8b%e5%ae%9a%e4%b9%89%e9%80%9f%e6%9f%a5%e8%a1%a8)
```
Comment (piotrwitek) [inline README_SOURCE-zh.md]: Here is a problem, this TOC is autogenerated using markdown headers because highlighted anchors don't have corresponding sections below they will be removed. You could fix this by adding markdown headers below and they will be added automatically by the script.

```doctoc --maxlevel=3 README_SOURCE.md README_SOURCE-zh.md```

you can find diff on my branch: https://github.com/piotrwitek/react-redux-typescript-guide/commit/f0c15a5494a38de856b21488977d3a935714730b


Basically the problems are that you should use Section headers as a source of truth NOT the table of contents which is autogenerated

Comment (seognil) [reply README_SOURCE-zh.md]: okay~
(I already found this problem days ago caused by my markdown editor plugins…)
I'd check that again with `doctoc` command is days ~

**Comment by seognil** [comment]
toc seems fixed

I'm also working on scripts for internationalization as you mention before

check this https://github.com/seognil-contributor/react-redux-typescript-guide/tree/feat-generate-readme

**Comment by piotrwitek** [comment]
That looks great, although I was thinking about something better. We could use mdx and some internationalization library, then extract all the text nodes that needs translations into a separate json file.
It should be easier to keep at sync when the guide is extended with a new content and also translating to other languages.

**Comment by seognil** [comment]
wow, you are right, i18n are some serious staff now!
really need to lookup in long run
